typescript/api-client/projects/vcd/schematics/src/schematics/plugin-seed/templates/9.7/__name@dasherize__/src/main/subnav-plugin.module.ts

```diff
@@ -0,0 +1,48 @@
+import { CommonModule } from "@angular/common";
```

Comment:
I know that the version param that you're checking is distinct, but isn't the 10.0 template exactly the same as the 9.7 template? Can both params just point to the same template folder?